jobs/haproxy/spec

```diff
@@ -114,3 +118,28 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
+  keepalived.vip:
+    description: VIP that will be given to master
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
+    default: check_haproxy
+  keepalived.healthcheck_command:
+    description: check that will be performed
+    default: killall -0 haproxy
+  keepalived.interface:
+    description: interface keepalived will use to mount the VIP
+    default: eth0
+  keepalived.masterip:
+    description: IP of masternode
+  keepalived.vrrp_instance_name:
+    description: name of the VRRP instance
+  keepalived.master_priority:
+    description: master weight
+    default: 101
+  keepalived.slave_priority:
+    description: slave weight
+    default: 100
+  keepalived.lvs_id:
+    description : Keepalived process identifier
```

Comment:
After running a few tests, spec only retrieves properties of the running job. It means that you can't access data like deployment name or bosh director.
I tried accessing meta.environment variable in the erb file but BOSH crashed. I suggest we use spec.name as default value and inform users in documentation.